oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDescendantSpellcheckCommonTest.java

```diff
@@ -38,4 +40,23 @@ protected Repository createJcrRepository() {
         return jcr.createRepository();
     }
 
+    //TODO ES Failing: this seems to be similar to
+    //https://discuss.elastic.co/t/elasticsearch-suggestion-completes-return-incorrect-number-of-suggestions/214522
+    //Even increasing the suggester query size the number of max results seems to be fixed to 5
+    @Test
+    @Ignore("Failing on ES")
```

Comment:
We often use the Jira issue id in the ignore annotation. So what about opening a new Oak Jira issue to fix this particular problem, and then use the id here? E.g. @Ignore("OAK-123456")

Reply:
I have created issues for all the failing tests and referenced them on the `@Ignore` tag.